src/Windowing/Silk.NET.Windowing.Common/Interfaces/IViewProperties.cs

```diff
@@ -54,6 +55,8 @@ public interface IViewProperties
         /// <summary>
         /// Whether or not VSync is enabled for this view.
         /// </summary>
+        /// <remarks>This value will set the SwapInterval for OpenGL, but on other frameworks (e.g. Vulkan) you will need to turn on
+        /// VSync manually.  Whilst <see langword="true"/>, the value of <see cref="FramesPerSecond"/> is ignored, to prevent frame skipping.</remarks>

```

Comment:
```suggestion
        /// <remarks>This value will set the SwapInterval for OpenGL, but on other frameworks (e.g. Vulkan) you will need to turn on
        /// VSync manually using that framework. Whilst <see langword="true"/>, the value of <see cref="FramesPerSecond"/> is ignored, to prevent frame skipping.</remarks>

```